### PR TITLE
Implemented button logics in codtrek applicant evaluate page

### PR DESCRIPTION
**Pull request by Priyanka-Kaintura**
Targets #3234

### Description

Implemented button logic for codeTrek applicant evaluate page.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by LavishRawat** [comment]
@Priyanka-Kaintura 
**Testing Feedbacks**
1) The Buttons are not aligned properly.Please have a look.

    
image

2) The `Start Internship` will only appear if the applicant is in `onboarded` round while for other cases it should not appear for `Active` tab.

image

**Comment by Priyanka-Kaintura** [comment]
Sir, I don't understand any use of accordion class here as tI cannot find anything to show in the item. Do we really need to keep this? If the divisions in the file are corrected the output looks something like this. 
image
image

**Comment by Priyanka-Kaintura** [comment]
@LavishRawat @Ayush8923 Sir, Is there any relevance of accordion in this page as all the items target the same data of the applicant. 
image

**Comment by Priyanka-Kaintura** [comment]
I think showing the applicant information only once at the top will be enough other than that only feedbacks can be made collapsable items. if it is suitable please let me know & I will try implementing it then.

**Comment by LavishRawat** [comment]
@Priyanka-Kaintura 
**Testing Feedbacks**
1) There is no space between the cards except for the gap between the first and second card.
image

2) The card footer is still visible even if the card is not expanded. Same happening in case of `InActive` applicant.
**Active**
    image
**Inactive**
    image

3) In the case of `Completed` applicant card footer should not be visible as there are no buttons to display.
    image

**Comment by LavishRawat** [comment]
@Priyanka-Kaintura 
**Testing Ui Feedback** 
1) Please align the `move to round`  field with the `Update Feedback ` field.

   image
   
2) Please along the ` Mark Active ` button with the `Update Feedback` field.
    image
    
    
 else the flow and functionality is fine.

**Comment by Priyanka-Kaintura** [comment]
Sir, do you mean to shift all the elements of footer to the right or just the margins in the footer.

**Comment by LavishRawat** [comment]
> Sir, do you mean to shift all the elements of footer to the right or just the margins in the footer.

Hey @Priyanka-Kaintura The feedback was to align the `move to round ` field with the fields above it.
    image

in your case , the following were not aligned.

image

**Comment by Priyanka-Kaintura** [comment]
@LavishRawat Sir, please check it now

File: Modules/CodeTrek/Resources/views/evaluate.blade.php [hunk 1]
```diff
@@ -62,33 +62,32 @@ class="collapse {{ $applicantDetail->id ? ($loop->last ? 'show' : '') : '' }}"
...
+                                @csrf
+                                    <div class="d-flex align-items-center">
+                                        <select name="round" id="rounds" class="w-22p">
+                                            @foreach (config('codetrek.rounds') as $round)
+                                                <option value="{{ $round['slug'] }}">Move to {{ $round['label'] }}
+                                                </option>
+                                            @endforeach
+                                        </select>
+                                        <button type="submit" class="btn btn-success ml-2">Take Action</button>
+                                    </div>
```
Comment (GauravGusain98) [inline Modules/CodeTrek/Resources/views/evaluate.blade.php:77]:
```suggestion
                                @csrf
                                <div class="d-flex align-items-center">
                                    <select name="round" id="rounds" class="w-22p">
                                        @foreach (config('codetrek.rounds') as $round)
                                            <option value="{{ $round['slug'] }}">Move to {{ $round['label'] }}
                                            </option>
                                        @endforeach
                                    </select>
                                    <button type="submit" class="btn btn-success ml-2">Take Action</button>
                                </div>
```

File: Modules/CodeTrek/Resources/views/evaluate.blade.php [hunk 2]
```diff
@@ -62,33 +62,32 @@ class="collapse {{ $applicantDetail->id ? ($loop->last ? 'show' : '') : '' }}"
...
-                                         </select>
-                                         <button type="submit" class="btn btn-success ml-2">Take Action</button>
-                             </form>
-                             <form action="{{ route('codetrek.updateStatus', $applicant->id) }}" method="POST">
-                                 @csrf
-                                 <button type="submit" name="action" value="completed" class="btn btn-dark ml-2">Mark
-                                     Completed</button>
-                                <button type="submit" name="action" value="{{ $applicant->status == "active"? "inactive" : "active" }}" class="btn btn-{{ $applicant->status == "active" ? "danger" : "info" }} ml-1">
+                        @if ($loop->last)
+                        <div class="card-footer row {{($applicant->status == 'completed') ? 'd-none': '' }} ml-1">
```
Comment (GauravGusain98) [inline Modules/CodeTrek/Resources/views/evaluate.blade.php:66]: There is a white gap coming on left side of footer. Please fix it
image

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

**Review by LavishRawat — APPROVED** [review]

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**